addons/mail/models/mail_alias.py

```diff
@@ -227,9 +229,13 @@ def open_parent_document(self):
         }
 
     def _get_alias_bounced_body_fallback(self, message_dict):
-        return _("""Hi,<br/>
-The following email sent to %s cannot be accepted because this is a private email address.
-Only allowed people can contact us at this address.""", self.display_name)
+        return Markup(
+            _("""<p>Hi,<br/>
+The following email sent to %(alias_display_name)s cannot be accepted because this is a private email address.
+Only allowed people can contact us at this address.</p>""",
+              alias_display_name=self.display_name
+             )
+        )
```

Comment:
Bit iffy to do the formatting inside the Markup call (rather than on it), here `mail.alias` is only C/U by system users but eeeh.

Reply:
Better to get formatting outside ?

Reply:
In this case it should not matter too much due to the ACL, but in general yes: here the parameter being formatted in will be treated as "safe markup", so if it comes from an untrusted party you've got yourself an XSS.

Reply:
Ok Sir.